src/FESpaces/DivConformingFESpaces.jl

```diff
@@ -0,0 +1,171 @@
+# This source file is though to put that code required in order to
+# customize ConformingFESpaces.jl to H(div)-conforming global FE Spaces built
+# out of RaviartThomas FEs. In particular, this customization is in the
+# definition of the shape functions (get_cell_shapefuns) and the DoFs
+# (get_cell_dof_basis) of the **global** FE space, which requires a sign flip
+# for those sitting on facets of the slave cell of the facet.
+
+# Two key ingredients in the implementation of this type of ReferenceFE are the
+# get_cell_shapefuns(model,cell_reffes,::Conformity) and
+# get_cell_dof_basis(mode,cell_reffes,::Conformity) overloads.
+# These are written such that, for each cell K, they return the shape functions
+# and dof values in the *global* RT space. For a dof owned by a face which is shared by
+# two cells, there is a master and a slave cell. The slave cell first computes the
+# shape functions and dof values using local-to-cell data structures, but then flips the
+# sign of both in order to get their corresponding counterparts in the **global**
+# RT space. As as result we have the following:
+
+# * When we interpolate a function into the global FE space, and we perform the cell-wise
+#   DoF values to global DoF values gather operation, we can either extract the global DoF value
+#   from the master or slave cell without worrying about the sign.
+# * When we evaluate a global FE function, and we perform the global DoF values to
+#   cell-wise DoF values scatter operation, we don't have to worry about the sign either.
+#   On the slave cell, we will have both the sign of the DoF value, and the sign of the
+#   shape function corresponding to the global DoF.
+# * We do NOT have to use the signed determinant, but its absolute value, in the Piola Map.
+
+function get_cell_dof_basis(model::DiscreteModel,
```

Comment:
It looks that when using RT FEs with L2 conformity, the default cell shape funs will be generated (i.e. the ones without  the transformation).   Is this enough?

Reply:
Yes this is enough because no conformity is to be imposed